modules/statics/src/coins.ts

```diff
@@ -6390,15 +6392,17 @@ export const coins = CoinMap.fromCoins([
     'USD Coin',
     6,
     '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-    UnderlyingAsset.USDC
+    UnderlyingAsset.USDC,
+    TOKEN_FEATURES_WITH_SWISS
   ),
   erc20(
     'e1d609f1-eb3f-4b3f-8ef6-87f434a6de83',
     'usdt',
     'Tether',
     6,
     '0xdac17f958d2ee523a2206206994597c13d831ec7',
-    UnderlyingAsset.USDT
+    UnderlyingAsset.USDT,
+    TOKEN_FEATURES_WITH_SWISS
```

Comment:
@mukeshsp  if we add another region, this naming will become irrelevant.
How we handle this ? can you find a better way ?

Reply:
In that case, we will add this feature with new variable similar to this case 
https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts#L260